syntax/diff.sublime-syntax

```diff
@@ -0,0 +1,66 @@
+%YAML 1.2
+---
+# http://www.sublimetext.com/docs/3/syntax.html
+name: GitSavvy Diff
+hidden: true
+scope: git-savvy.diff
+first_line_match: |-
+  (?x)^
+      (===\ modified\ file
+      |==== \s* // .+ \s - \s .+ \s+ ====
+      |Index:[ ]
+      |---\ [^%]
+      |\*\*\*.*\d{4}\s*$
+      |\d+(,\d+)* (a|d|c) \d+(,\d+)* $
+      |diff\ --git[ ]
+      )
+contexts:
+    - match: '(^(-{3}) a/(.+)$\n?)'
+      comment:
```

Comment:
I think YAML expects multi-line values to have something here denoting as much.  I'm guessing that nothing broke with this comment, otherwise you would have fixed it.  But could you change this to `      comment: |`, just so it is consistent?